src/main/java/com/mraof/minestuck/util/StartingModusManager.java

```diff
@@ -0,0 +1,56 @@
+package com.mraof.minestuck.util;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonElement;
+import com.google.gson.reflect.TypeToken;
+import com.mraof.minestuck.Minestuck;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.server.packs.resources.ResourceManager;
+import net.minecraft.server.packs.resources.SimpleJsonResourceReloadListener;
+import net.minecraft.util.profiling.ProfilerFiller;
+import net.minecraftforge.event.AddReloadListenerEvent;
+import net.minecraftforge.eventbus.api.SubscribeEvent;
+import net.minecraftforge.fml.common.Mod;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+@Mod.EventBusSubscriber(bus = Mod.EventBusSubscriber.Bus.FORGE)
+public class StartingModusManager extends SimpleJsonResourceReloadListener
+{
+	private static final Gson GSON = new GsonBuilder().registerTypeAdapter(BoondollarPricing.class, new BoondollarPricing.Serializer()).create();
+	
+	private static StartingModusManager INSTANCE;
+	private List<String> startingModusTypes;
+	
+	public StartingModusManager()
+	{
+		super(GSON, "minestuck/config");
+	}
+	
+	@Override
+	protected void apply(Map<ResourceLocation, JsonElement> jsonEntries, ResourceManager resourceManager, ProfilerFiller profiler)
+	{
+		JsonElement element = jsonEntries.get(new ResourceLocation(Minestuck.MOD_ID, "starting_modus"));
+		this.startingModusTypes = GSON.fromJson(element, new TypeToken<ArrayList<String>>(){}.getType());
```

Comment:
It may be appropriate to parse this into a list of `ModusType`s, to take that responsibility away from `PlayerData`, and also to log an error earlier when some list entry cannot be parsed into a modus type.